addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -72,7 +72,9 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
         text-align: center;
 
         img {
-            max-height: 100%;
+            height: 100%;
+            width: 100%;
+            object-fit: scale-down;
```

Comment:
What does this do?

Reply:
It allows the product image to fit its container, so the user can click everywhere on the frame to replace it. The spec has been added by sbu and the fix has been found by sad,
I will add that to the commit message

Reply:
Ok, the `<img/>` tag is expanded but the size of the seen image is the same right?

Reply:
exactly !